aperturedb/Entities.py

```diff
@@ -39,19 +40,43 @@ def retrieve(cls,
         Returns:
             List[Entities]: _description_
         """
+
+        # Sice adjacent items are usually a way to filter the results,
+        # the native query is constructed in the reverse order, with
+        # first filtering out the relavant itmes based on adjacent items.
```

Comment:
```suggestion
        # Since adjacent items are usually a way to filter the results,
        # the native query is constructed in the reverse order, 
        # first filtering out the relevant items based on adjacent items.
```